draft-ietf-quic-multipath.md

```diff
@@ -1399,7 +1399,40 @@ TBD (experiments use 0x1001d76d3ded42f3)| MP_PROTOCOL_VIOLATION | Multipath prot
 
 # Security Considerations
 
-TBD
+## Path States
+
+As the maximum Path ID is negotiated by both endpoints with transport parameter initial_max_path_id
+and MAX_PATH_ID frame, Endpoints can limit the usage of Path ID and associated path resources by
+limiting the maximum Path ID issued by themselves.
+
+Before creating a new path, endpoints MUST issue unused CIDs for the corresponding Path ID.
+Endpoints also have a strategy to only issue new CIDs of the lowest available unused Path ID
+for limiting the creation for new paths.
+
```

Comment:
Not sure about the capital MUST here. We are not making a protocol specification, but merely explaining that endpoints can control the number of paths.